app/views/keywords/_upload-csv-form.html.slim

```diff
@@ -1,3 +1,10 @@
-= form_with(model: keyword, url: keywords_path) do |form|
-  = form.file_field :keywords_file, accept: 'text/csv', required: true, multipart: true
-  = form.submit 'Submit'
+.keywords-form
+  = form_with(model: keyword, url: keywords_path) do |form|
+    = render 'shared/flash_messages'
+
+    .keywords-form__upload
+      p Upload a CSV File
+      = form.file_field :keywords_file, accept: 'text/csv', required: true, multipart: true
+
+    .keywords-form__submit
+      = form.submit 'Submit', class: 'keywords-form__button'
```

Comment:
```suggestion
.keywords-form
  = form_with(model: keyword, url: keywords_path, class: 'keywords-form__form') do |form|
    = render 'shared/flash_messages'

    .keywords-form__upload
      p.keywords-form__placeholder Upload a CSV File
      = form.file_field :keywords_file, accept: 'text/csv', class: 'keywords-form__input-file', required: true, multipart: true

    = form.submit 'Submit', class: 'keywords-form__submit-button'
```

How about this?
And for the style will be like this.

```scss
.keywords-form {
  display: flex;
  flex-direction: column;
  margin-top: rem(20px);
  
  &__form {
    @include neumorphism-block;
    display: flex;
    position: relative;
    flex-direction: column;
    max-width: rem(450px);
    transition: $basic-transition;
    border-radius: rem(10px);

    background-color: $white-100;

    &:hover {
      border-top: 3px solid $pink;
    }
  }
  
  &__placeholder {
    font-style: italic;
  }
  
  &__upload {
    padding: rem(15px);
  }
  
  &__input-file {
    argin-bottom: rem(20px);
  }
  
  &__submit-button {
    display: flex;
    margin-top: auto;
    width: 100%;
    height: 30px;
    border: unset;
    background-color: $pink;
    color: $white-100;
  }
}
```

Reply:
breaking down the inside component will reduce the depth of the selector 🤩
![Screen Shot 2565-10-19 at 13 51 57](https://user-images.githubusercontent.com/749672/196617820-a791bd79-cdcf-4a7d-8a1d-2cdf0e714c79.png)
ref: https://github.com/nimblehq/compass/blob/develop/_docs/development/code-conventions/css.md

Reply:
Yup I like this, it also follows Nimble's conventions 👷🏾 Updated in https://github.com/aetheryna/google-scraper-ruby/pull/35/commits/f347b499d2460bbd6f358239317f5953a1edd419